### DISCUSSION TITLE
[BUG] Convert the classic SharePoint page to modern SharePoint page

### DISCUSSION THREAD

**Discussion by gustavorost** [pnp/powershell#3202] [General]
### Expected behavior 
My expectation is to convert the classic SharePoint page to modern SharePoint page

### Actual behavior 
I can´t convert the page using the ConvertTo-PnPPage command

### Steps to reproduce behavior
I´m using the custom code below that I created using PnP Framework

$SiteURL = "https://my-domain.sharepoint.com/sites/my_site"
 
Connect-PnPOnline $SiteURL -Interactive
 
$Pages = Get-PnPListItem -List SitePages

Try {
    ForEach ($Page in $Pages) {
        $PageName = $Page.FieldValues.FileLeafRef
        Write-host "Converting Page:"$PageName
 
        #Check if the page is classic
        If ($Page.FieldValues["ClientSideApplicationId"] -eq "b6917cb1-93a0-4b97-a84d-7cf49975d4ec") {
            Write-host "`tPage is already Modern:"$PageName -f Yellow
        }
        Else {
            #Conver the classic page to modern page
            ConvertTo-PnPPage -Identity $PageName -Overwrite -TakeSourcePageName -AddPageAcceptBanner
            Write-host "`tPage Converted to Modern!" -f Green    
        }
    }
}
Catch {
    write-host -f Red "Error Converting Clasic Page to Modern!" $_.Exception.Message
}

When I run this command I receive the error message below

"Error Converting Clasic Page to Modern! 'I' is an invalid start of a property name. Expected a '"'. Path: $ | LineNumber: 0 | BytePositionInLine: 628."

Anyone knows how we can fix this issue or to convert a classic basic page to the modern page?

### What is the version of the Cmdlet module you are running?
1.12.0     PnP.PowerShell

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by veronicageek** [comment]
@gustavorost - Please update the module to the latest (you'll need PS `7.2` minimum), or try other pages as well, and let us know.

**Reply by gustavorost** [reply]
@veronicageek I have updated my PowerShell to 7.3.5 version but the error it still happening.

I have tried to do in the other pages but without success.

Any other idea?

**Reply by ZombaFlo** [reply]
@gustavorost Did you fix it?

**Reply by gdnhsg** [reply]
This still happens intermittently to SP2016 pages during conversion. It appears to affect pages with locally referenced assets, like images.

**Comment by ZombaFlo** [comment]
Same issue here :(

**Comment by ToddKlindt** [comment]
I get the same error with one page I'm trying to convert. Also from SharePoint 2016. I'm using PnP.PowerShell version 2.99.117.

**Comment by rlv-dan** [comment]
We had this problem with images. Solved it by using the `-SkipUrlRewriting` flag.
